### PR TITLE
refactor: bench readme and rename license file

### PR DESCRIPTION
**Pull request by Thunderbottom**

### CODE REVIEW

**Comment by kennethsequeira** [comment]
@gavindsouza review and merge.

File: README.md [hunk 1]
```diff
@@ -1,131 +1,236 @@
...
-- This is an opinionated setup so it is best to setup on a blank server.
-- Works on Ubuntu 16.04+, CentOS 7+, Debian 8+
-- You may have to install Python 3 and other essentials by running `apt-get install python3-minimal build-essential python3-setuptools`
-- This script will install the pre-requisites, install bench and setup an ERPNext site `(site1.local under frappe-bench)`
-- Passwords for Frappe Administrator and MariaDB (root) will be asked and saved under `~/passwoords.txt`
-- MariaDB (root) password may be `password` on a fresh server
-- You can then login as **Administrator** with the Administrator password
-- The log file is saved under `/tmp/logs/install_bench.log` in case you run into any issues during the install.
-- If you find any problems, post them on the forum: [https://discuss.erpnext.com](https://discuss.erpnext.com/c/bench) with the `installation_problem` under "Install / Update" category.
+### Manual Installation
```
Comment (gavindsouza) [inline README.md:154]: add direct bench install command `$ pip install frappe-bench` and summary of system dependencies

File: README.md [hunk 2]
```diff
@@ -1,131 +1,236 @@
 <div align="center">
 	<img src="https://github.com/frappe/design/raw/master/logos/png/bench-logo.png" height="128">
-	<h2>bench</h2>
+	<h2>Bench</h2>
 </div>
 
-bench is a command-line utility that helps you to install apps, manage multiple sites and update Frappe / ERPNext apps on */nix (macOS, Ubuntu, Debian, CentOS, etc) for development and production.
+Bench is a command-line utility that helps you to install, update, and manage multiple sites for Frappe/ERPNext applications on [*nix systems](https://en.wikipedia.org/wiki/Unix-like) for development and production.
```
Comment (gavindsouza) [inline README.md:6]:
```suggestion 
bench is a command-line utility that helps you to install, update, and manage multiple sites for Frappe/ERPNext applications on [*nix systems](https://en.wikipedia.org/wiki/Unix-like) for development and production.
```

Comment (Thunderbottom) [reply README.md:6]: i really dont think it should be lowercase. its a command line utility, i get that, but i dont see the point in capitalization.

File: README.md (hunk 1 above)
Comment (Thunderbottom) [reply README.md:154]: the linked docs have a list of all dependencies

**Comment by revant** [comment]
https://frappe.io/docs/user/en/installation

this also needs change

**Comment by Thunderbottom** [comment]
which has already been done at the same time with https://github.com/frappe/frappe_io/pull/271

File: README.md (hunk 1 above)
Comment (gavindsouza) [reply README.md:154]: Add a reference to `$ pip install frappe-bench` for advanced users who can set up their system manually. Maybe suggesting an increasing order of difficulty for setting up a Frappe environment.

File: README.md (hunk 2 above)
Comment (gavindsouza) [reply README.md:6]: The reason for de-capitalization was that it's a command-line utility. It's 'Bench' if you believe that's what it should be.

**Pull request merged**